refactor_me.py

```diff
@@ -3,24 +3,35 @@
 from collections import namedtuple, defaultdict
 from operator import itemgetter
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
-def sum_expenses(expenses, min_amount=0):
-    aggregated_expenses = defaultdict(int)
-    for expense in expenses:
-        amount = expense.amount
-        if amount >= min_amount:
-            aggregated_expenses[expense.type_] += amount
-    return aggregated_expenses
+class BaseExpencesManager:
+    _buffer = ()
 
+    def add(self, expenses):
+        for expense in expenses:
+            self._buffer += (expense, )
 
-def print_expenses(expenses):
-    for expense, amount in sorted(expenses.items(), key=itemgetter(1)):
-        print(expense, amount)
+    def by_category(self, treshold=0):
+        # do not include expenses less than treshold
+        aggregated_expenses = defaultdict(int)
+        for expense in self._buffer:
+            if expense.amount >= treshold:
+                aggregated_expenses[expense.category] += expense.amount
+        return aggregated_expenses
+
+
+class ExpencesManager(BaseExpencesManager):
+    @staticmethod
+    def report(expenses):
+        for category, amount in sorted(expenses.items(), key=itemgetter(1)):
+            print(f"{category}: {amount}")
```

Comment:
be consistent about using single and double quotes for string literals across the code base, I prefer single quotes.